scalafix-nsc/src/test/scala/scalafix/ImportPatches.sc

```diff
@@ -0,0 +1,23 @@
+// Small worksheet to demonstrate usage of import patches
```

Comment:
@ShaneDelmore I got this working on my laptop
![screen shot 2017-01-28 at 21 30 54](https://cloud.githubusercontent.com/assets/1408093/22399633/1333d81c-e5a1-11e6-91d6-8fcb189b2876.png)